src/diagnostics.ts

```diff
@@ -128,6 +122,32 @@ export class Diagnostics {
     await Lifecycle.getInstance().emit('Doctor:diagnostic', { testName, status });
   }
 
+  public async networkCheck(): Promise<void> {
```

Comment:
is something supposed to call this method?

Reply:
it's called by the doctor, any methods ending in `Check` are called